lib/ubus.c

```diff
@@ -1087,25 +1092,36 @@ uc_ubus_request_reply(uc_vm_t *vm, size_t nargs)
 	uc_ubus_request_t **callctx = uc_fn_this("ubus.request");
 	int64_t code = UBUS_STATUS_OK;
 	uc_value_t *reply, *rcode;
+	bool more = false;
 
 	if (!callctx || !*callctx)
 		err_return(UBUS_STATUS_INVALID_ARGUMENT, "Invalid call context");
 
 	args_get(vm, nargs,
-	         "reply", UC_OBJECT, true, &reply,
-	         "rcode", UC_INTEGER, true, &rcode);
+	         "reply", UC_OBJECT, OPTIONAL, &reply,
+	         "rcode", 0, OPTIONAL, &rcode);
```

Comment:
Maybe it would make sense to introduce a new virtual status code for this purpose, e.g. `UBUS_STATUS_CONTINUE` or `UBUS_STATUS_MORE`. It could use code value `-1` which would not clash with potential future status code additions to libucode.

The resulting code would look slightly more explicit:
```javascript
req.reply({ blah: 1 }, UBUS_STATUS_CONTINUE);
req.reply({ blah: 2 }, UBUS_STATUS_CONTINUE);
req.reply({ blah: 3 }, UBUS_STATUS_OK);
```